src/xmipp/libraries/reconstruction/volume_subtraction.cpp

```diff
@@ -185,26 +220,67 @@ class ProgVolumeSubtraction: public XmippProgram
     	mask1.clear();
 		POCSmask(mask(),V1());
     	POCSnonnegative(V1());
-
     	double v1min, v1max;
 		V1().computeDoubleMinMax(v1min, v1max);
 
-    	transformer1.FourierTransform(V1(),V1Fourier,false);
-    	FFT_magnitude(V1Fourier,V1FourierMag);
-		double std1 = V1().computeStddev();
-
 		V.read(fnVol2);
 		POCSmask(mask(),V());
     	POCSnonnegative(V());
 
+		// Compute |FT(radial averages)|
+    	MultidimArray<double> V1rad, Vrad;
+    	V1rad = V1();
+    	Vrad = V();
+    	V1rad.setXmippOrigin();
+    	Vrad.setXmippOrigin();
+		FourierTransformer transformerRad;
+		MultidimArray< std::complex<double> > V1FourierRad, VFourierRad;
+		MultidimArray<double> V1FourierMagRad, VFourierMagRad, radQuotient;
```

Comment:
extract a method that would do the work of line 237 through 244 (returning `VFourierMagRad`)